model/load_balancer.rb

```diff
@@ -0,0 +1,34 @@
+#  frozen_string_literal: true
+
+require_relative "../model"
+
+class LoadBalancer < Sequel::Model
+  many_to_many :vms
+  one_to_one :strand, key: :id
+  many_to_one :private_subnet
+  one_to_many :projects, through: :private_subnet
+
+  include ResourceMethods
+  include SemaphoreMethods
+  include Authorization::TaggableMethods
+  include Authorization::HyperTagMethods
+  semaphore :destroy, :update_load_balancer
+
+  def hyper_tag_name(project)
+    "project/#{project.ubid}/load-balancer/#{name}"
+  end
+
+  def add_vm(vm)
```

Comment:
should we name this `add_vm` and `remove_vm` or `attach_vm` and `detach_vm`?

Reply:
I have named them as `add_vm`, `remove_vm`, `evacuate_vm`. Is this good?

Reply:
```
    # In crockford's base32 multiple characters can map to a single number,
    # for example ["1","I","L"] all map to 1. However, the reverse (number
    # -> character) only uses one canonical one (e.g. 1 maps to "1").
    # This test ensures we only use canonical characters in our type
    # constants, so ids are actually prefixed by those constants.
    #
    # See https://www.crockford.com/base32.html
```